src/sparseml/pytorch/sparsification/quantization/helpers.py

```diff
@@ -571,15 +572,57 @@ def get_observer(
     qconfig_kwargs: Dict[str, Any],
 ):
     qscheme = torch.per_tensor_symmetric if symmetric else torch.per_tensor_affine
-    quant_min, quant_max = compute_range(dtype, bits)
+    quant_min, quant_max, is_custom_qrange = compute_range(dtype, bits)
+
+    observer_cls = torch_quantization.MovingAverageMinMaxObserver
     observer_kwargs = dict(
-        observer=torch_quantization.MovingAverageMinMaxObserver,
-        quant_min=quant_min,
-        quant_max=quant_max,
         dtype=dtype,
         qscheme=qscheme,
         reduce_range=reduce_range,
     )
+
+    # in torch 1.9.1, quant_min and quant_max are not passed to observer:
```

Comment:
For readability, why not put it between triple quotation marks? Will it be picked up by the doc parsing? If not, I'd like that, instead of commenting separate lines out. for readability.

Reply:
I am almost sure your first sentence is correct (triple quotes only get picked up if they appear in the docstring context)

Reply:
Updated